libmscore/rest.cpp

```diff
@@ -532,7 +532,7 @@ int Rest::computeLineOffset(int lines)
                   }
             }
 
-      if (offsetVoices) {
+      if (offsetVoices && voice() < 2) {
```

Comment:
This means voice 1 and 2, right? IIRC `voice()` is 0 based.

Reply:
Yes, don't offset voice 1 and voice 2 in combination with slash notation, but do offset voice 3 and 4

Reply:
Might be worth a comment explaining this?

Reply:
For detailed description please see https://musescore.org/en/node/318800#comment-1074063

Reply:
Yes, that's where I found it. Might be good though to summarize this in a short comment in the code